win32_deps_build.sh

```diff
@@ -29,8 +29,8 @@ boostSrcDir="${depsSrcDir}/boost_1_70_0"
 boostDir="${depsToolsetDir}/boost"
 zlibDir="${depsToolsetDir}/zlib"
 zlibSrcDir="${depsSrcDir}/zlib"
-backtraceDir="${depsToolsetDir}/backtrace"
-backtraceSrcDir="${depsSrcDir}/backtrace"
+backtraceDir="${depsToolsetDir}/libbacktrace"
```

Comment:
win32_build.sh will also need to be updated to use this path.

Reply:
Ah, good catch! Sorry for missing that one.